resources/ansible/network_contacts.yml

```diff
@@ -0,0 +1,9 @@
+---
+- hosts: all
+  become: False
+  tasks:
+    - name: copy network contacts locally
+      fetch:
+        src: /home/{{ ansible_user }}/node_data/section_tree
```

Comment:
`section_tree` does not exist these days. Might just be updaating a var name?

`network_contacts`?

Reply:
Thanks for noticing this. I think we can just get rid of this whole playbook now.